README.md

```diff
@@ -29,41 +29,25 @@ Key features:
 
 ## Installation:
 
-### Using pre-built libraries
```

Comment:
we still want to preserve the option to install via libraries. IGN_GAZEBO_SYSTEM_PLUGIN_PATH is deprecated, so use GZ_SIM_SYSTEM_PLUGIN_PATH instead and instead of IGN_GUI_PLUGIN_PATH use GZ_GUI_PLUGIN_PATH

Reply:
Could you create a release for the libraries just like the in the fortress branch

Reply:
[RGLGazeboPlugin_ubuntu22(Garden).zip](https://github.com/RobotecAI/RGLGazeboPlugin/files/12504346/RGLGazeboPlugin_ubuntu22.Garden.zip)
Here is the zip for the release